### DISCUSSION TITLE
[WARNING] You have 3 previously executed migrations in the database that are not  registered migrations.

### DISCUSSION THREAD

**Discussion by Gamzo20** [symfony/symfony#47272] [Symfony Book: The Fast Track]
Hello 
please help me solve this problem, I'm using symfony 6, then when I want to make a migration to my database and  I got this problem: **[WARNING] You have 3 previously executed migrations in the database that are not  registered migrations.**, 
So I want to know how can I register  the 3 previous migrations . thank you

**Comment by stof** [comment]
You need to go back to the branch were those migration classes exist in the codebase (and then migrate them down)
